README.md

```diff
@@ -10,13 +10,14 @@ _Another hobby stat site for another Halo game_
 1. PHP7.4 or PHP8
 2. Node + Yarn installed
 3. [Composer](https://getcomposer.org/) installed.
-4. `cp .env.example .env`
```

Comment:
I think removing the `cp .env.example .env` was too much. We probably need that still.